src/TurbulenceClosures/vertically_implicit_diffusion_solver.jl

```diff
@@ -180,16 +180,20 @@ function implicit_step!(field::Field,
     if closure isa Tuple
         closure_tuple = closure
         N = length(closure_tuple)
-        vi_closure = Tuple(closure[n] for n = 1:N if is_vertically_implicit(closure[n]))
+        vi_closure            = Tuple(closure[n]            for n = 1:N if is_vertically_implicit(closure[n]))
         vi_diffusivity_fields = Tuple(diffusivity_fields[n] for n = 1:N if is_vertically_implicit(closure[n]))
+
+        # length(vi_closure) > 1 && error("More than one closure with VerticallyImplicitTimeDiscretization is not supported")
```

Comment:
remove or is it still the case?

Reply:
Hmm, it's not tested and I suspect CATKE may not work with a tuple yet. But I think we should save for a future PR, this PR does enough already. I'll remove the comment

Reply:
Sorry let me clarify -- we test tupled closures with implicit vertical diffusion and they pass (I also tested myself and found that it works). CATKE is special though, I think there is a bug with the linear component (only CATKE has this). But I'd like to debug and implement a test in another PR.